CRITICAL - 2022-04-07 04:42:09 --> Object of class App\Controllers\Submit could not be converted to string
#0 C:\xampp\htdocs\pait\system\CodeIgniter.php(832): App\Controllers\Submit->admin()
#1 C:\xampp\htdocs\pait\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Submit))
#2 C:\xampp\htdocs\pait\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\pait\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 C:\xampp\htdocs\pait\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#5 {main}
CRITICAL - 2022-04-07 04:48:51 --> Object of class App\Controllers\Submit could not be converted to string
#0 C:\xampp\htdocs\pait\system\CodeIgniter.php(832): App\Controllers\Submit->admin()
#1 C:\xampp\htdocs\pait\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Submit))
#2 C:\xampp\htdocs\pait\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\pait\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 C:\xampp\htdocs\pait\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#5 {main}
CRITICAL - 2022-04-07 04:50:33 --> Object of class App\Controllers\Submit could not be converted to string
#0 C:\xampp\htdocs\pait\system\CodeIgniter.php(832): App\Controllers\Submit->admin()
#1 C:\xampp\htdocs\pait\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Submit))
#2 C:\xampp\htdocs\pait\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\pait\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 C:\xampp\htdocs\pait\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#5 {main}
CRITICAL - 2022-04-07 05:03:28 --> There is no data to insert.
#0 C:\xampp\htdocs\pait\system\BaseModel.php(718): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('insert')
#1 C:\xampp\htdocs\pait\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#2 C:\xampp\htdocs\pait\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#3 C:\xampp\htdocs\pait\app\Controllers\Submit.php(32): CodeIgniter\BaseModel->save(Array)
#4 C:\xampp\htdocs\pait\system\CodeIgniter.php(832): App\Controllers\Submit->admin()
#5 C:\xampp\htdocs\pait\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Submit))
#6 C:\xampp\htdocs\pait\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\pait\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\pait\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#9 {main}
CRITICAL - 2022-04-07 05:06:58 --> There is no data to insert.
#0 C:\xampp\htdocs\pait\system\BaseModel.php(718): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('insert')
#1 C:\xampp\htdocs\pait\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#2 C:\xampp\htdocs\pait\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#3 C:\xampp\htdocs\pait\app\Controllers\Submit.php(32): CodeIgniter\BaseModel->save(Array)
#4 C:\xampp\htdocs\pait\system\CodeIgniter.php(832): App\Controllers\Submit->admin()
#5 C:\xampp\htdocs\pait\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Submit))
#6 C:\xampp\htdocs\pait\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\pait\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\pait\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#9 {main}
CRITICAL - 2022-04-07 05:11:43 --> There is no data to insert.
#0 C:\xampp\htdocs\pait\system\BaseModel.php(718): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('insert')
#1 C:\xampp\htdocs\pait\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#2 C:\xampp\htdocs\pait\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#3 C:\xampp\htdocs\pait\app\Controllers\Submit.php(32): CodeIgniter\BaseModel->save(Array)
#4 C:\xampp\htdocs\pait\system\CodeIgniter.php(832): App\Controllers\Submit->admin()
#5 C:\xampp\htdocs\pait\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Submit))
#6 C:\xampp\htdocs\pait\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\pait\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\pait\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#9 {main}
CRITICAL - 2022-04-07 07:08:47 --> Call to undefined method CodeIgniter\Database\MySQLi\Connection::insert_id()
#0 C:\xampp\htdocs\pait\app\Controllers\Submit.php(26): App\Models\UserMDL->saveUser()
#1 C:\xampp\htdocs\pait\system\CodeIgniter.php(832): App\Controllers\Submit->admin()
#2 C:\xampp\htdocs\pait\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Submit))
#3 C:\xampp\htdocs\pait\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\pait\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\pait\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2022-04-07 07:11:41 --> Undefined property: App\Controllers\Submit::$db
#0 C:\xampp\htdocs\pait\app\Controllers\Submit.php(26): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined prope...', 'C:\\xampp\\htdocs...', 26)
#1 C:\xampp\htdocs\pait\system\CodeIgniter.php(832): App\Controllers\Submit->admin()
#2 C:\xampp\htdocs\pait\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Submit))
#3 C:\xampp\htdocs\pait\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\pait\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\pait\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2022-04-07 07:14:55 --> Undefined property: App\Controllers\Submit::$db
#0 C:\xampp\htdocs\pait\app\Controllers\Submit.php(28): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined prope...', 'C:\\xampp\\htdocs...', 28)
#1 C:\xampp\htdocs\pait\system\CodeIgniter.php(832): App\Controllers\Submit->admin()
#2 C:\xampp\htdocs\pait\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Submit))
#3 C:\xampp\htdocs\pait\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\pait\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\pait\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2022-04-07 07:20:45 --> Cannot access protected property CodeIgniter\Database\MySQLi\Builder::$db
#0 C:\xampp\htdocs\pait\system\CodeIgniter.php(832): App\Controllers\Submit->admin()
#1 C:\xampp\htdocs\pait\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Submit))
#2 C:\xampp\htdocs\pait\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\pait\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 C:\xampp\htdocs\pait\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#5 {main}
CRITICAL - 2022-04-07 07:24:47 --> Cannot access protected property CodeIgniter\Database\MySQLi\Builder::$db
#0 C:\xampp\htdocs\pait\app\Controllers\Submit.php(27): App\Models\UserMDL->lastID()
#1 C:\xampp\htdocs\pait\system\CodeIgniter.php(832): App\Controllers\Submit->admin()
#2 C:\xampp\htdocs\pait\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Submit))
#3 C:\xampp\htdocs\pait\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\pait\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\pait\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2022-04-07 07:36:04 --> There is no data to update.
#0 C:\xampp\htdocs\pait\system\BaseModel.php(1574): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
#1 C:\xampp\htdocs\pait\system\BaseModel.php(832): CodeIgniter\BaseModel->transformDataToArray(NULL, 'update')
#2 C:\xampp\htdocs\pait\system\Model.php(651): CodeIgniter\BaseModel->update(Array, NULL)
#3 C:\xampp\htdocs\pait\app\Controllers\Submit.php(36): CodeIgniter\Model->update(Array)
#4 C:\xampp\htdocs\pait\system\CodeIgniter.php(832): App\Controllers\Submit->admin()
#5 C:\xampp\htdocs\pait\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Submit))
#6 C:\xampp\htdocs\pait\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\pait\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\pait\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#9 {main}
CRITICAL - 2022-04-07 08:01:53 --> There is no data to update.
#0 C:\xampp\htdocs\pait\system\BaseModel.php(1574): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
#1 C:\xampp\htdocs\pait\system\BaseModel.php(832): CodeIgniter\BaseModel->transformDataToArray(NULL, 'update')
#2 C:\xampp\htdocs\pait\system\Model.php(651): CodeIgniter\BaseModel->update(Array, NULL)
#3 C:\xampp\htdocs\pait\app\Controllers\Submit.php(38): CodeIgniter\Model->update(Array)
#4 C:\xampp\htdocs\pait\system\CodeIgniter.php(832): App\Controllers\Submit->admin()
#5 C:\xampp\htdocs\pait\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Submit))
#6 C:\xampp\htdocs\pait\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\pait\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\pait\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#9 {main}
CRITICAL - 2022-04-07 08:03:15 --> There is no data to update.
#0 C:\xampp\htdocs\pait\system\BaseModel.php(1574): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
#1 C:\xampp\htdocs\pait\system\BaseModel.php(832): CodeIgniter\BaseModel->transformDataToArray(NULL, 'update')
#2 C:\xampp\htdocs\pait\system\Model.php(651): CodeIgniter\BaseModel->update(Array, NULL)
#3 C:\xampp\htdocs\pait\app\Controllers\Submit.php(38): CodeIgniter\Model->update(Array)
#4 C:\xampp\htdocs\pait\system\CodeIgniter.php(832): App\Controllers\Submit->admin()
#5 C:\xampp\htdocs\pait\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Submit))
#6 C:\xampp\htdocs\pait\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\pait\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\pait\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#9 {main}
CRITICAL - 2022-04-07 08:06:18 --> There is no data to update.
#0 C:\xampp\htdocs\pait\system\BaseModel.php(1574): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
#1 C:\xampp\htdocs\pait\system\BaseModel.php(832): CodeIgniter\BaseModel->transformDataToArray(NULL, 'update')
#2 C:\xampp\htdocs\pait\system\Model.php(651): CodeIgniter\BaseModel->update(Array, NULL)
#3 C:\xampp\htdocs\pait\app\Controllers\Submit.php(41): CodeIgniter\Model->update(Array)
#4 C:\xampp\htdocs\pait\system\CodeIgniter.php(832): App\Controllers\Submit->admin()
#5 C:\xampp\htdocs\pait\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Submit))
#6 C:\xampp\htdocs\pait\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\pait\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\pait\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#9 {main}
CRITICAL - 2022-04-07 15:58:20 --> Too few arguments to function App\Controllers\Admin::index(), 0 passed in C:\xampp\htdocs\pait\system\CodeIgniter.php on line 832 and exactly 1 expected
#0 C:\xampp\htdocs\pait\system\CodeIgniter.php(832): App\Controllers\Admin->index()
#1 C:\xampp\htdocs\pait\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
#2 C:\xampp\htdocs\pait\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\pait\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 C:\xampp\htdocs\pait\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#5 {main}
CRITICAL - 2022-04-07 16:53:21 --> Undefined array key "status"
#0 C:\xampp\htdocs\pait\app\Views\admin\administrator.php(52): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array...', 'C:\\xampp\\htdocs...', 52)
#1 C:\xampp\htdocs\pait\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\pait\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\pait\system\Common.php(1089): CodeIgniter\View\View->render('admin/administr...', Array, true)
#4 C:\xampp\htdocs\pait\app\Controllers\Admin.php(44): view('admin/administr...', Array)
#5 C:\xampp\htdocs\pait\system\CodeIgniter.php(832): App\Controllers\Admin->index('user')
#6 C:\xampp\htdocs\pait\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
#7 C:\xampp\htdocs\pait\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\pait\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\pait\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
ERROR - 2022-04-07 16:59:40 --> Unknown column 'status' in 'field list'
CRITICAL - 2022-04-07 16:59:40 --> Unknown column 'status' in 'field list'
#0 C:\xampp\htdocs\pait\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\pait\system\Database\BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\pait\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\pait\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\pait\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\pait\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\pait\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\pait\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\pait\app\Controllers\Submit.php(31): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\pait\system\CodeIgniter.php(832): App\Controllers\Submit->admin()
#10 C:\xampp\htdocs\pait\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Submit))
#11 C:\xampp\htdocs\pait\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\pait\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\pait\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#14 {main}
